### PR TITLE
[WIP] Refactor HTML/CSS through the Community page

### PR DESCRIPTION
**Pull request by zeppelin**
This PR aims to add responsive layout to the website, but a lot of refactoring is planned before in order to do that.

### CODE REVIEW

**Comment by acorncom** [comment]
@zeppelin Would suggest we do this in small increments (and multiple PRs) instead of one giant PR.  Due to how visible changes here could be, we're all a bit leery of major PRs :smile: 

I'm at the contributors workshop if you want to discuss in more detail

**Comment by zeppelin** [comment]
Sure, I'll update the PR's title & branch name to reflect the actual (smaller) changes. The current title is only meant to communicate the end goal.

**Comment by toovy** [comment]
@zeppelin In parallel I've also started with responsive adaptations. I've started with the about page, introduced bourbon/neat as grid, mostly just reworked CSS and some JS/templates to get the examples running. I did not really scratch the HTML, but the refactor you are doing is probably also important. For the about page the only thing left is the Tomster Mail example. It heavily relies on tables...

If we join forces somehow I think we could get it done faster. How shall we proceed? Can I create a PR to your fork?

File: source/stylesheets/community.css.scss [hunk 1]
```diff
@@ -0,0 +1,107 @@
+@import "compass";
+@import "mixins/hidpi";
+
+@mixin community-content($selector) {
```
Comment (mixonic) [inline source/stylesheets/community.css.scss]: Why use a mixin?

**Comment by zeppelin** [comment]
@toovy sounds awesome, definitely interested in joining forces! I'm back home now from EmberConf, will check out your branch & concept, as soon as I catch up with the rest of my things.

**Comment by toovy** [comment]
@zeppelin nice, I'll see if I can push my results today!

**Comment by locks** [comment]
@zeppelin @toovy @ryanlabouve any updates?

**Comment by toovy** [comment]
@locks @zeppelin reviews my stuff today (or so) with the aim to create a development strategy afterwards.

**Comment by locks** [comment]
@toovy @zeppelin how did the review go?

**Comment by toovy** [comment]
@locks no feedback yet. Don't know how to proceed.

**Comment by zeppelin** [comment]
@toovy @locks I'm alive! Had no time whatsoever to work on OSS - so I did quit the company I was working for :) I'm reviewing it right now. Might take a some though...

**Comment by toovy** [comment]
@zeppelin wow, cool stuff. I'm not available for the next two weeks so pls feel free to merge/rewrite my PR.

**Comment by locks** [comment]
I am closing this PR due to lack of movement. Please keep the changes smaller next time, it makes it easier to review and merge. And we don't mind multiple PRs :]

**Comment by zeppelin** [comment]
@locks you're quite right about closing it. The commits are there and ready to be cherry-picked whenever I'm able to continue working on it. However, I'm intend to bring the website & the guides closer together, this should make the responsive movement easier.

**Pull request closed without merging**